services/common/src/constants/actionTypes.js

```diff
@@ -187,3 +187,6 @@ export const CLEAR_TAILINGS_STORAGE_FACILITY = "CLEAR_TAILINGS_STORAGE_FACILITY"
 
 // Dams
 export const STORE_DAM = "STORE_DAM";
+
+//Verifiable Credentials
+export const STORE_VC_WALLET_CONNECTION_INVITATION = "STORE_VC_WALLET_CONNECTION_INVITATION";
```

Comment:
Good catch.  Added this in my current work as well.  I think that was my bad that this got lost in the shuffle of moving stuff around.

Reply:
I'm hoping this can be merged and deployed to DEV/TEST. Minespace VC invitations won't show on the screen until this is in.